src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -4566,4 +4566,65 @@ public LazyWithStatement(Lazy<IOperation> body, Lazy<IOperation> value, bool isI
         public override IOperation Value => _lazyValue.Value;
     }
 
+    /// <summary>
+    /// Represents a local function statement.
+    /// </summary>
+    internal abstract partial class BaseLocalFunctionStatement : Operation, ILocalFunctionStatement
+    {
+        protected BaseLocalFunctionStatement(IMethodSymbol localFunctionSymbol, bool isInvalid, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
```

Comment:
I have used the common pattern already used in this generated file.

---
In reply to: [121588312](https://github.com/dotnet/roslyn/pull/20177#discussion_r121588312) [](ancestors = 121588312)